web/pages/product.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react'
+import axios from 'axios'
+import ProductContainer from '../components/product/ProductContainer.jsx'
+import { useParams } from 'react-router-dom'
+
+const Product = () => {
+  const [product, setProduct] = useState(null)
+  const { id } = useParams()
+
+  useEffect(() => {
+    async function fetchProduct() {
+      try {
+        const response = await axios.get(
```

Comment:
we already have code to fetch product details in product service. Can we copy it from v1 into here and re-use it?

Reply:
You mean copy the code/file?
> "server\services\product.service.js" is this the file?

Reply:
no I think it's `public/v1/J's/services/product-service.js`